spec/unit/chunker_spec.rb

```diff
@@ -69,16 +96,45 @@ def @throttler.stride
                                                            :start     => 1,
                                                            :limit     => 1)
 
+      @connection.expects(:select_value).with(regexp_matches(/where id >= 1 order by id limit 1 offset 0/)).returns(nil)
```

Comment:
I don't understand why this returns `nil` when the update on the next line matches? It's between 1 and 1, but not >=1 ?